provisioner/buildinfo.go

```diff
@@ -15,8 +15,7 @@ type BuildInfo struct {
 	PackerRunUUID  string `json:"PackerRunUUID"`
 	Password       string `json:"Password"`
 
-	SSHAgentAuth      bool   `json:"SSHAgentAuth"`
-	SSHPrivateKey     string `json:"SSHPrivateKey"`
-	SSHPrivateKeyFile string `json:"SSHPrivateKeyFile"`
```

Comment:
According to the [docs](https://www.packer.io/docs/templates/legacy_json_templates/engine) there is no `SSHPrivateKeyFile` so I removed it